sys/ztimer/auto_init.c

```diff
@@ -107,18 +108,20 @@ ztimer_clock_t *const ZTIMER_USEC = &_ztimer_convert_frac_usec.super.super;
 #if MODULE_ZTIMER_MSEC
 #  if MODULE_PERIPH_RTT
```

Comment:
I would change this to `#if MODULE_ZTIMER_PERIPH_RTT`. This should be the correct way to check whether `ZTIMER_MSEC` should be backed by `periph_rtt`.

Reply:
After a closer look, I would say this change is more related to #13967.